src/config/aws-post-config.ts

```diff
@@ -74,6 +79,31 @@ Without management account credentials ${CLIName} won't be able to execute the c
     writeConfig(config);
   }
 
+  private async configureAccountAccessRole(config: Config) {
+    if (config.aws.accountAccessRole) {
+      console.log(
+        `There was an AWS account access role defined: '${config.aws.accountAccessRole}'. To change this role run '${CLICommand} config aws'`,
+      );
+      return;
+    }
+
+    console.log(
+      `There is no AWS account access role selected. You must set a role with access to the account you want to scan. This role usually is called the 'OrganizationAccountAccessRole'.`,
+    );
+
+    // Prompt the user with the found profiles.
+    const selectedRole: string = await Input.prompt({
+      message:
+        "Choose an AWS CLI profile with access to the management account",
```

Comment:
```suggestion
        "Write a RoleName that you can access from your management account with AssumeRole method",
```

Reply:
changed it but adapted it as maybe `RoleName` is a bit too specific as we usually did not use so "tech specific" language. But I changed it to: `Specify a Role that you can access from your management account with the AssumeRole method:`